src/main/webapp/app/app.main.ts

```diff
@@ -6,9 +6,12 @@ ProdConfig();
 
 if (module['hot']) {
     module['hot'].accept();
+    if ('production' !== process.env.NODE_ENV) {
+        console.clear();
```

Comment:
Why do we need this code?

Reply:
this was a helper for me that clears the console when there is a hot reload. I often looked at errors that did not came from the current build but from older one. this code prevents this issue. but i can revert it again and just add it locally for me

Reply:
ok, I guess we can leave it in then